scripts/eud-tendermint-node2.sh

```diff
@@ -0,0 +1,28 @@
+#!/usr/bin/env bash
```

Comment:
Can we add all Tendermint-specific scripts into the same folder including maybe a quick explanation of what each of them do? Thanks!

Reply:
Fixed